### ISSUE TITLE
Use a half-press gesture to perform a click

### ISSUE THREAD

**Issue by javifernandez** [Igalia/wolvic#1487]
**Is your feature request related to a problem? Please describe.**
I have to fully press down on the primary trigger with my meta quest 3 controllers in order to press a button or key. This is slow, causes me to miss button presses.

**Describe the solution you'd like**
In contrast to the Oculus browser where I only have to press halfway.

**Describe alternatives you've considered**

**Additional context**

**Comment by svillar** [comment]
I don't think we are changing anything here. The actual threshold we use to consider a press as a click is 91% and it has been working fine for most if not all devices. Changing that to a lower value would result in tons of new unsolicited click and/or drag events.

**Comment by javifernandez** [comment]
If you compare with the browser shipped by Meta, the pressure level is considerable lower there.
